WriteVHDLSyntax.py

```diff
@@ -128,7 +128,7 @@ def writeTopLevelMemoryInstance(memModule, interface):
                     wirelist += "std_logic_vector(3 downto 0);\n"
             else:
                 wirelist += "  signal "+memModule.inst+"_nentries_"+str(i)+"_V_dout : "
-                wirelist += "std_logic_vector(7 downto 0);\n"
+                wirelist += "std_logic_vector(6 downto 0);\n"
```

Comment:
If this line is changing, we should also change L117 and L212, correct?

Reply:
Ah, I completely missed that. I didn't change them because those lines don't exist in Ians PR. Do I still change them or wait until Ians branch gets merged and just ignore it...? (I don't know what the best git procedures are... :-) )

Reply:
Okay now I understand, those ports are no longer needed because they correspond to nentries coming out of HLS modules and going into the intermediate memories. So the best thing to do is ignore it - if you change it now there will be merge conflicts when we try to merge Ian's PR.